aria_plugin/executor.py

```diff
@@ -12,24 +12,65 @@
 #    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #    * See the License for the specific language governing permissions and
 #    * limitations under the License.
-
+import os
+import pickle
+import subprocess
+import sys
+import tempfile
 from threading import Thread
 
 
-from cloudify import ctx
+# This is a hacky way of patching the ruamel before trying to load aria stuff.
+import aria_plugin                                              # noqa: F401
+
 from aria.orchestrator import workflow_runner
+from aria.orchestrator.workflows.executor import process
 from aria.cli.logger import ModelLogIterator
 
-from .environment import Environment
 
+# This is a very hacky way of executing some code before loading any aria
+# related code (more specifically patching the ruamel.yaml issue.
+class ARIAPluginExecutor(process.ProcessExecutor):
+    def _execute(self, ctx):
+        # This is a a copy of the Process Executor _execute. we do that since
+        # we need to execute ruamel patching before the execution but inside
+        # the subprocess.
+
+        # The file to execute should be configurable, and not automatically
```

Comment:
Remove this. If you want, create a Jira issue about this in ARIA, and in there, think of the consequences. But this is indeed an edge case.